app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -40,14 +40,14 @@ class MainActivity : ComponentActivity() {
 }
 
 @Composable
-fun Greeting(name: String) {
+fun greeting(name: String) {
     Text(text = "Hello $name!")
 }
 
 @Preview(showBackground = true)
 @Composable
-fun DefaultPreview() {
+fun defaultPreview() {
```

Comment:
Changed function to camel case.
![functionNaming](https://user-images.githubusercontent.com/60935069/156949039-e2a80341-4765-40e2-a0f8-7d87111d06da.PNG)

Reply:
Yeah... except composables.  And I think that's stupid.  :(

https://developer.android.com/kotlin/style-guide